### PR TITLE
README: Replace instructions with link to docs

### PR DESCRIPTION
**Pull request by dcarley**
The docs are far more complete and appear to be more actively
maintained than this README. With the following exceptions:

- no link to circleci/realitycheck; maybe they should?
- no teardown instructions; so I've retained those
- no documentation for all variables; these seem of limited value since
  they duplicate information in `variables.tf` and haven't been
  maintained either (e.g. #61)

### CODE REVIEW

**Comment by MikeyYeahYeah** [comment]
1) Interested in getting feedback from the SE team.
2) I still think it's worth pointing people to the variables.  So maybe rather than gutting that section entirely, we could point out that the variables could be found there.  We should, at the very, least maintain the descriptions for the variables.

**Comment by dcarley** [comment]
> Interested in getting feedback from the SE team.

Do they watch this repo? Do we need to ask someone explicitly?

> I still think it's worth pointing people to the variables.  So maybe rather than gutting that section entirely, we could point out that the variables could be found there.  We should, at the very, least maintain the descriptions for the variables.

Good point. I've rebased to add a link to `variables.tf`.

**Review by demophoon — APPROVED** [review]

**Comment by demophoon** [comment]
@rosieyohannan We've went ahead and rebased this to point to the website for all documentation. If this looks good to you can you approve this PR for us?
Thanks

**Comment by rosieyohannan** [comment]
@demophoon Just a couple of things but I don't have write access so can't push a change... The link to the old install guide in the first paragraph can be removed (whole sentence can be removed), and might be good to add a, ##Upgrade section after ##Installation to point to the upgrade guide: https://circleci.com/docs/2.0/updating-server/#section=server-administration?

**Review by yugesh84 — APPROVED** [review]
Added Rosie so she can confirm, but LGTM.

**Review by ryanwohara — APPROVED** [review]
I feel like a necromancer even looking at a PR this old

**Comment by robinsturmeit** [comment]
> Added Rosie so she can confirm, but LGTM.

Rosie's on PTO, I think. I'll leave it as is until she is able to take a look

**Comment by dcarley** [comment]
> I feel like a necromancer even looking at a PR this old

😆 👻

File: README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
+This package allows you to easily orchestrate your CircleCI Server cluster in AWS using Terraform.
 
 **Note: This is only meant to be used for the initial setup of CircleCI Server and is not meant to be used for the ongoing maintenance of the CircleCI Server.**
 
 **Note: Master is the only supported branch. All other branches of this repo should not be considered stable, and is to be used at your own risk.**
 
-# Getting Started
-
-## Pre Reqs
+## Documentation
```
Comment (rosieyohannan) [inline README.md:9]: Might be worth pointing people to the Prerequisites and Planning doc as well the install guide? https://circleci.com/docs/2.0/aws-prereq/#section=server-administration. This details needing terraform, and all the other bits of info they are going to need to have to hand.

File: README.md [hunk 2]
```diff
@@ -73,3 +39,12 @@ Optional vars:
   | services_user_data_enabled | Enable/disable automated installation on Services Box | true |
   | force_destroy_s3_bucket | Add/Remove ability to forcefully destroy S3 bucket | false |
   | services_disable_api_termination | Protect the services instance from API termination | true |
+
+
+### Teardown
+
+You can find teardown instructions at https://circleci.com/docs/2.0/aws-teardown/
```
Comment (rosieyohannan) [inline README.md:46]:
```suggestion
You can find teardown instructions at https://circleci.com/docs/2.0/aws-teardown/.
```

File: README.md [hunk 3]
```diff
@@ -18,44 +16,12 @@ We use Terraform to automate parts of the infrastructure for your CircleCI Serve
...
-1. First you need to manually disable the termination protection on the Services machine from the AWS Management Console (If you set `services_disable_api_termination = "false"` in `terraform.tfvars`, skip this step.). To do this:
-
-    1. Navigate to the EC2 Dashboard and locate the Services machine instance
-    1. Click to select it
-    1. Click Actions > Instance Settings > Change Termination Protection
-
-
-1. Navigate to the S3 dashboard, locate the S3 bucket associated with your CircleCI cluster and delete it/its contents (If you set `force_destroy_s3_bucket = "true"` in `terraform.tfvars`, skip this step.).
-1. From a terminal, navigate to your clone of our `enterprise-setup` repo and run `terraform destroy` to destroy all EC2 instances, IAM roles, ASGs and Launch configurations created by `terraform apply`.
+You can find instructions here: https://circleci.com/docs/2.0/aws/
```
Comment (rosieyohannan) [inline README.md:19]:
```suggestion
You can find instructions here: https://circleci.com/docs/2.0/aws/.
```

File: README.md [hunk 4]
```diff
@@ -73,3 +39,12 @@ Optional vars:
...
   | services_disable_api_termination | Protect the services instance from API termination | true |
+
+
+### Teardown
+
+You can find teardown instructions at https://circleci.com/docs/2.0/aws-teardown/
+
+## Upgrade
+
+If you want to upgrade from a previous installation, please follow these [upgrade instructions](https://circleci.com/docs/2.0/updating-server/#section=server-administration)
```
Comment (rosieyohannan) [inline README.md:50]:
```suggestion
If you want to upgrade an existing installation from a previous version, follow these [upgrade instructions](https://circleci.com/docs/2.0/updating-server/#section=server-administration).
```

**Comment by rosieyohannan** [comment]
Just a few suggestions :-)

**Comment by rosieyohannan** [comment]
Sorry I just pressed 'close' by mistake 😱

File: README.md (hunk 1 above)
Comment (robinsturmeit) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (robinsturmeit) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -1,61 +1,23 @@
...
+## Prerequisites and Requirements
 
-## Pre Reqs
-
-We use Terraform to automate parts of the infrastructure for your CircleCI Server install, so you will need to install this first:
-
-* [Terraform](https://www.terraform.io/downloads.html)
-
-**Note: This script only supports terraform version 0.12 and higher. Please update to the most recent version before fetching from upstream.**
+Please refer to our prerequisites documentation here: https://circleci.com/docs/2.0/aws-prereq
```
Comment (rosieyohannan) [inline README.md:11]:
```suggestion
Please refer to our prerequisites documentation here: https://circleci.com/docs/2.0/aws-prereq.
```

File: README.md [hunk 6]
```diff
@@ -1,61 +1,23 @@
 # CircleCI Server Setup
 
-This package allows you to easily orchestrate your CircleCI Server cluster in AWS using Terraform. For a full step by step guide to installing CircleCI Server with Terraform, see our [installation guide](https://circleci.com/docs/2.0/circleci-install-doc-v2-17.pdf#section=administration).
+This package allows you to easily orchestrate your CircleCI Server cluster in AWS using Terraform.
```
Comment (rosieyohannan) [inline README.md:3]:
```suggestion
This package allows you to orchestrate your CircleCI Server cluster in AWS using Terraform.
```

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**